viewer/packages/camera-manager/src/Flexible/FlexibleCameraManager.ts

```diff
@@ -0,0 +1,551 @@
+    } else if (mouseActionType === FlexibleMouseActionType.SetTargetAndCameraPosition) {
+      if (this.controls.controlsType !== FlexibleControlsType.Orbit) {
+        this.controls.setControlsType(FlexibleControlsType.Orbit);
+      }
+      this.setTargetAndCameraPosition(event);
+    }
+  }
+
+  private async setTargetAndCameraPosition(event: PointerEventData) {
+    const raycastResult = await this._raycastCallback(event.offsetX, event.offsetY, true);
+    // If an object is picked, zoom in to the object (the target will be in the middle of the bounding box)
+    if (raycastResult.pickedBoundingBox !== undefined) {
+      const { position, target } = fitCameraToBoundingBox(this.camera, raycastResult.pickedBoundingBox, 3);
+      moveCameraTo(this, position, target, this.options.animationDuration);
+      return;
+    }
+    // If not particular object is picked, set camera position half way to the target
+    const newTarget =
+      raycastResult.intersection?.point ??
+      this.getTargetByBoundingBox(event.offsetX, event.offsetY, raycastResult.modelsBoundingBox);
+
+    const newPosition = new Vector3().subVectors(newTarget, this.camera.position);
+    newPosition.divideScalar(2);
+    newPosition.add(this.camera.position);
+    moveCameraTo(this, newPosition, newTarget, this.options.animationDuration);
+  }
+
+  //================================================
+  // INSTANCE METHODS: Updates
+  //================================================
+
+  private updateCameraNearAndFar(): void {
+    // See https://stackoverflow.com/questions/8101119/how-do-i-methodically-choose-the-near-clip-plane-distance-for-a-perspective-proj
+    if (this._isDisposed) {
+      return;
+    }
+    if (!this.options.automaticNearFarPlane) {
+      return;
+    }
+    CameraManagerHelper.updateCameraNearAndFar(this.camera, this._currentBoundingBox);
+  }
+
+  private updateControlsSensitivity(): void {
+    if (this._isDisposed) {
+      return;
+    }
+    if (!this.options.automaticSensitivity) {
+      return;
+    }
+    // This is used to determine the speed of the camera when flying with ASDW.
+    // We want to either let it be controlled by the near plane if we are far away,
+    // but no more than a fraction of the bounding box of the system if inside
+    const diagonal = this.getBoundingBoxDiagonal2();
+    const diagonalFraction = diagonal * this.options.sensitivityDiagonalFraction;
+    const nearFraction = 0.1 * this.camera.near;
+
+    let sensitivity: number;
+    if (diagonalFraction < nearFraction) {
+      sensitivity = nearFraction;
+    } else {
+      sensitivity = this.options.getLegalSensitivity(diagonalFraction);
+    }
+    this.options.sensitivity = sensitivity;
+
+    // Nils: Need this for a while to fine tune
+    // console.log('diagonalFraction:     ', diagonalFraction);
+    // console.log('nearFraction:         ', nearFraction);
+    // console.log('sensitivity:          ', sensitivity);
```

Comment:
Can you make a commit where you remove this, and then add them back to your local code? I really don't want things like this hanging around